subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/ValidatePlugins.java

```diff
@@ -43,6 +43,11 @@
 /**
  * Validates plugins by checking property annotations on work items like tasks and artifact transforms.
  *
+ * This task should be added to a Gradle plugin project for doing static analysis on the plugin classes.
```

Comment:
Maybe reverse the logic and say that the `java-gradle-plugin` adds this task by default _but_ that if you're in the unusual case of not using this plugin you need to register it yourself.

Reply:
Done.